modules/ROOT/pages/database-administration/aliases/naming-aliases.adoc

```diff
@@ -0,0 +1,27 @@
+:description: Rules for naming database aliases in Neo4j.
+[role=enterprise-edition aura-db-enterprise]
+= Naming rules for database aliases
+
+Database alias names are subject to the standard Cypher restrictions on valid identifiers.
```

Comment:
- [ ] this sentence is now both here and in the note, I think it's enough to have it up here?
- [ ] in the database naming section it is only in the note but I think I prefer having it up here instead

Reply:
Fixed for both.